src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -110,19 +147,22 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
       <PageTitle title="New Change Request" previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]} />
       <PageBlock title="Details">
         <Grid container spacing={2}>
-          <Grid item xs={12} md={3}>
+          <Grid item xs={12} md={6}>
             <Box>
-              <Typography variant="caption">WBS Number</Typography>
+              <Typography sx={{ ml: 1 }} variant="caption">
+                WBS
+              </Typography>
```

Comment:
can you change these typographies to formlabels inside of formcontrols while you're here (see Ari's PR on those create forms)

Reply:
i think that is out of scope and also a good easy ticket to make

Reply:
Sounds good